worksheet.md

```diff
@@ -131,6 +131,6 @@ Some of the cool features on the micro:bit are the GPIO pins, the accelerometer,
 - As always, the best way of finding out how to use a new piece of hardware or software (in this case both) is to have a look at the documentation. You can find documentation for MicroPython on the micro:bit at [this site](https://microbit-micropython.readthedocs.org/en/latest/).
 - You could have a go at some of the other cool projects using a Raspberry Pi and a micro:bit. Have a look at the following resources:
 1. [micro:bit selfies](https://www.raspberrypi.org/learning/microbit-selfies)
-2. [micro:bit interactive badge](https://projects.raspberrypi.org/en/projects/interactive-badge)
+2. [micro:bit interactive badge](https://www.raspberrypi.org/learning/interactive-badge)
```

Comment:
This URL is giving me a 404?

Reply:
Does it need to use the ‘projects’ url for that one - same as above step_10 file

Reply:
Replaced with `projects` site url, learning is not working